src/ucs/sys/sys.c

```diff
@@ -1565,9 +1590,29 @@ ucs_status_t ucs_sys_get_memlock_rlimit(size_t *rlimit_value)
     *rlimit_value = (limit_info.rlim_cur == RLIM_INFINITY) ?
                             SIZE_MAX :
                             limit_info.rlim_cur;
+
     return UCS_OK;
 }
 
+ucs_status_t ucs_sys_get_effective_memlock_rlimit(size_t *rlimit_value)
+{
+    int mlock_cap = 0;
+
```

Comment:
Move blank line below (separate vars decl from other code).

Reply:
`mlock_cap` is already separated. 
do you mean also separate `ucs_status_t status`?

Reply:
```
int mlock_cap = 0;
ucs_status_t status = ucs_sys_get_mlock_cap(&mlock_cap);

if (status != UCS_OK) {
...
```